lib/ansible/modules/cloud/vmware/vmware_content_library_manager.py

```diff
@@ -0,0 +1,261 @@
+---
+module: vmware_content_library_manager
+short_description: Create, Update and delete content library
+description:
+- Module to create content Library
+- Content Library feature is introduced in vSphere 6.0 version, so this module is not supported in the earlier versions of vSphere.
+- All variables and VMware object names are case sensitive.
+version_added: '2.9'
+author:
+- Pavan Bidkar (@pgbidkar)
+notes:
+- Tested on vSphere 6.5, 6.7
+requirements:
+- python >= 2.6
+- PyVmomi
+- vSphere Automation SDK
+options:
+    library_name:
+      description:
+      - The name of library_name to create.
+      type: str
+      required: True
+    library_description:
+      description:
+      - The content library description .
+      - This is required only if C(state) is set to C(present).
+      - This parameter is ignored, when C(state) is set to C(absent).
+      - Process of updating content library only allows description change.
+      type: str
+      required: False
+      default: ''
+    datastore_name:
+      description:
+      - Name of the datastore on which backing content library is created
+      - This is required only if C(state) is set to C(present).
+      - This parameter is ignored, when C(state) is set to C(absent).
+      - Currently only datastore backing creation is supported
+      type: str
+      required: False
+    state:
+      description:
+      - The state of content library.
+      - If set to C(present) and library does not exists, then tag is created.
+      - If set to C(present) and library exists, then tag is updated.
+      - If set to C(absent) and library exists, then tag is deleted.
+      - If set to C(absent) and library does not exists, no action is taken.
+      type: str
+      required: False
+      default: 'present'
+      choices: [ 'present', 'absent' ]
+extends_documentation_fragment: vmware_rest_client.documentation
+'''
+
+EXAMPLES = r'''
+- name: Create Content Library
+  vmware_content_library_manager:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    library_name: test-content-lib
+    library_description: 'Library with Datastore Backing'
+    datastore_name: datastore
+    validate_certs: False
+    state: present
+  delegate_to: localhost
+
+- name: Update Content Library
+  vmware_content_library_manager:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    library_name: test-content-lib
+    library_description: 'Library with Datastore Backing'
+    validate_certs: no
+    state: present
+  delegate_to: localhost
+
+- name: Delete Content Library
+  vmware_content_library_manager:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    library_name: test-content-lib
+    validate_certs: no
+    state: absent
+  delegate_to: localhost
+'''
+
+RETURN = r'''
+results:
```

Comment:
```suggestion
content_library_info:
```
or 
`library_info` would be more apt.